target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -37,6 +37,7 @@ ath79_setup_interfaces()
 	engenius,ecb1200|\
 	engenius,ecb1750|\
 	engenius,ecb600|\
+	engenius,emr3000-v2|\
```

Comment:
the device has 2 ethernet ports, but you have it in a block for only 1
it should be in the block with enstationac-v1